verification/testing/certifyKey.go

```diff
@@ -341,6 +344,10 @@ func checkCertifyKeyExtensions(t *testing.T, extensions []pkix.Extension, flags
 	checkCertifyKeyBasicConstraints(t, extensions, flags)
 	checkCertifyKeyExtendedKeyUsages(t, extensions, bc.IsCA)
 	checkCertifyKeyTcgUeidExtension(t, extensions, label)
+	if IsX509 {
+		checkCertifyKeySubjectKeyIdentifierExtension(t, extensions, bc.IsCA, pubkey)
+		checkCertifyKeyAuthorityKeyIdentifierExtension(t, extensions, bc.IsCA, pubkey)
```

Comment:
If I understand RFC 5280 correctly, I think the AKI is supposed to use the Issuer's public key

Reply:
Done, added a function to the platform to retrieve the issuing cert's SubjectKeyIdentifier.